src/js/providers/tracks-mixin.js

```diff
@@ -401,6 +401,18 @@ define(['../utils/underscore',
     ////// PRIVATE METHODS
     //////////////////////
 
+    function _addCueToTrack(track, vttCue) {
+        if (!utils.isEdge()) {
```

Comment:
@hghazzi `TextTrackCue` also exists in browsers with the `VTTCue` interface, so that wasn't a reliable check. We only need to use `TextTrackCue` in Edge. In IE11, we polyfill `VTTCue` and render with `VTT.js`.